addons/OPT/tracker/Events.hpp

```diff
@@ -1,31 +1,31 @@
-//Eventliste
-
-// Event der Ausgelöst wird, wenn ein OPT Karten Dialog geöffnet wird
-#define OPTKartenDialogOpen "OPTKartenDialogOpen"
-
-// Event der Ausgelöst wird, wenn ein BIS Artillery Dialog geöffnet wird
-#define BISArtilleryDialogOpen "BISArtilleryDialogOpen"
-
-// Event der Ausgelöst wird, wenn ein BIS UAV Dialog geöffnet wird
-#define BISUAVDialogOpen "BISUAVDialogOpen"
-
-// Event der einen Marker erstellt in Clib System 
-#define OPTAddMarker "OPTAddMarker"
-
-// Event der einen Text an einem Marker erstellt/wechselt
-#define OPTsetMarkerText "OPTsetMarkerText"
-
-// Event der einen Icon an einem Marker erstellt/wechselt
-#define OPTsetMarkerIcon "OPTsetMarkerIcon"
-
-// Event der einen Color an einem Marker erstellt/wechselt
-#define OPTsetMarkerColor "OPTsetMarkerColor"
-
-// Event der einen Hover Marker erstellt/wechselt
-#define OPTsetMarkerHover "OPTsetMarkerHover"
-
-// Event der einen Hover Marker löscht
-#define OPTremoveMarkerHover "OPTremoveMarkerHover"
-
-// Event der einen Marker löscht
-#define OPTremoveMarker "OPTremoveMarker"
\ No newline at end of file
+//Eventliste
+
+// Event der Ausgelöst wird, wenn ein OPT Karten Dialog geöffnet wird
+#define OPTKARTENDIALOGOPEN "OPTKartenDialogOpen"

```

Comment:
In der Uppercase Schreibweise werden die Leerzeichen meist mit einem Unterstrich ausgetauscht. Das macht den String sehr viel Lesbarer:
`#define KARTEN_DIALOG_OPEN "OPTKartenDialogOpen"`

Gilt auch für alle anderen Vorkomnisse.

Reply:
Am Besten sogar `EVENT_KARTEN_DIALOG_OPEN`